scripts/coverage-ci

```diff
@@ -10,14 +10,14 @@ cargo build --all-targets
 
 # Need to be kept in sync manually with tests we run inside check.yml.
 cargo test
-# disabled until 961 is merged
-# for gate in "compact" "descriptive"; do
-#  cargo test --no-default-features --features "cli web-app real-world-infra test-fixture $gate-gate"
-# done
-cargo test --no-default-features --features "cli web-app real-world-infra test-fixture descriptive-gate"
 
-cargo test --bench oneshot_ipa --no-default-features --features "enable-benches descriptive-gate" -- -n 62 -c 16
+# descriptive-gate is enabled by default
+for gate in "compact-gate" ""; do
+  cargo test --no-default-features --features "cli web-app real-world-infra test-fixture $gate"
+done
 
-cargo test --bench criterion_arithmetic --no-default-features --features "enable-benches descriptive-gate"
+cargo test --bench oneshot_ipa --no-default-features --features "enable-benches" -- -n 62 -c 16
```

Comment:
This might benefit from a comment:

```suggestion
# Test that our benchmark works when compact-gate is disabled
cargo test --bench oneshot_ipa --no-default-features --features "enable-benches" -- -n 62 -c 16
```